includes/datatypes.php

```diff
@@ -848,3 +848,73 @@ function get_podcasting_language_options() {
 		)
 	);
 }
+
+/**
+ * Validate Podcast Taxonomy Fields.
+ *
+ * @param string $term     Term.
+ * @param string $taxonomy Taxonomy Name.
+ * @param array  $args     List of arguments.
+ *
+ * @return string
+ */
+function validate_taxonomy_fields( $term, $taxonomy, $args = [] ) {
+	// Bailout, if not the podcasts taxonomy.
+	if ( 'podcasting_podcasts' !== $taxonomy ) {
+		return $term;
+	}
+
+	$referer              = sanitize_text_field( $_POST['_wp_http_referer'] );
+	$is_onboarding_step_1 = false !== strpos( $referer, 'page=simple-podcasting-onboarding&step=1' );
+
+	if ( ! $is_onboarding_step_1 && empty( trim( $term ) ) ) {
+		return new \WP_Error( 'empty_term_name', __( 'A podcast name is required.' ) );
```

Comment:
```suggestion
		return new \WP_Error( 'empty_term_name', __( 'A podcast name is required.', 'simple-podcasting' ) );
```

:1234: This applies to all of the translations in this function, all of them will need the domain added.

Reply:
Yes, https://github.com/10up/simple-podcasting/pull/216/commits/558766c998d0742634c5bcb967389e2a814c5c47